main.go

```diff
@@ -98,54 +99,76 @@ type config struct {
 }
 
 // RejectionCondition contains the fields that should match a bug report for it to be rejected.
+// The condition matches a specific app, with optionall version and label, or the user submitted text
 type RejectionCondition struct {
-	// Required field: if a payload does not match this app name, the condition does not match.
+	// If a payload is not a UserTextMatch and does not match this app name, the condition does not match.
 	App string `yaml:"app"`
 	// Optional: version that must also match in addition to the app and label. If empty, does not check version.
 	Version string `yaml:"version"`
 	// Optional: label that must also match in addition to the app and version. If empty, does not check label.
 	Label string `yaml:"label"`
+	// Alternative to matching app names, match the content of the user text
+	UserTextMatch       string `yaml:"usertext"`
+	UserTextMatchReason string `yaml:"reason"`
```

Comment:
can we make `reason`  more general: ie, make it apply even if `UserTextMatch` is not given? and if `reason` is unspecified, we make up something sensible depending on what conditions are specified?

Reply:
Yes. I think the approach you propose over will take care of this.